lib/winston-logstash.js

```diff
@@ -75,6 +74,7 @@ winston.transports.Logstash = Logstash;
 Logstash.prototype.name = 'logstash';
 
 Logstash.prototype.log = function (level, msg, meta, callback) {
+  meta = meta !== null && meta !== undefined && !(meta instanceof Error) ? cycle.decycle(meta) : meta || null;
```

Comment:
Agreed, winston does it already, but we kept having max stack exceeded errors...

Sorry for this shitty PR, I will recreate it by the end of the week, including a test on circular references.